docs/api-reference.md

```diff
@@ -155,22 +155,30 @@ Use care when logging so that you don’t expose sensitive information like pass
-| ----------------------- | ------------------------------------------------ |
-| attributeAuthentication | Connection attribute for the authentication type |
-| attributeClass          | Connection attribute for the connection type     |
-| attributeDatabase       | Connection attribute for the database            |
-| attributeInitialSQL     | Connection attribute for initial SQL             |
-| attributePassword       | Connection attribute for the password            |
-| attributePort           | Connection attribute for the port                |
-| attributeServer         | Connection attribute for the server              |
-| attributeService        | Connection attribute for the service             |
-| attributeSSLCert        | Connection attribute for the SSL Certfile        |
-| attributeSSLMode        | Connection attribute for the SSL Mode            |
-| attributeUsername       | Connection attribute for the user name           |
-| attributeWarehouse      | Connection attribute for the Warehouse           |
-| keywordODBCUsername     | ODBC Username keyword                            |
-| keywordODBCPassword     | ODBC Password keyword                            |
+| Function                       | Description                                                 |
+| ------------------------------ | ----------------------------------------------------------- |
+| attributeAuthentication        | Connection attribute for the authentication type            |
+| attributeClass                 | Connection attribute for the connection type                |
+| attributeDatabase              | Connection attribute for the database                       |
+| attributeInitialSQL            | Connection attribute for initial SQL                        |
+| attributePassword              | Connection attribute for the password                       |
+| attributePort                  | Connection attribute for the port                           |
+| attributeServer                | Connection attribute for the server                         |
+| attributeService               | Connection attribute for the service                        |
+| attributeSSLCert               | Connection attribute for the SSL Certfile                   |
+| attributeSSLMode               | Connection attribute for the SSL Mode                       |
+| attributeTableauServerAuthMode | Connection attribute for tableau server authentication mode |
+| attributeTableauServerUser     | Connection attribute for tableau server user                |
+| attributeUsername              | Connection attribute for the user name                      |
+| attributeWarehouse             | Connection attribute for the Warehouse                      |
+| keywordODBCUsername            | ODBC Username keyword                                       |
+| keywordODBCPassword            | ODBC Password keyword                                       |
+
+_Attribute values_
```

Comment:
I'm not sure we need a separate table for this. These are more string constants, and valueAuthModeDBImpersoante fits that well enough.